src/main/java/org/skroutz/elasticsearch/index/analysis/WordDelimiterFilter.java

```diff
@@ -595,6 +596,17 @@ private boolean has(int flag) {
     return (flags & flag) != 0;
   }
 
+  private boolean isAlphaNumeric(String term) {
+    char[] charArr = term.toCharArray();
+
+    for(char c : charArr) {
+      if(!Character.isLetter(c) && !Character.isDigit(c)) {
```

Comment:
We can combine those checks with [`isLetterOrDigit`](https://docs.oracle.com/javase/7/docs/api/java/lang/Character.html#isLetterOrDigit(char)).

Reply:
Done